gateway/src/resty/tls.lua

```diff
@@ -0,0 +1,91 @@
+local base = require "resty.core.base"
```

Comment:
> Please ignore codecov report. I will move tls.lua to apicast-nginx-module in the future commit.

Do we plan to move this file (and content) to the  apicast-nginx-module??

Reply:
Yes, I'm planing to replace https://github.com/3scale/apicast-nginx-module/blob/master/lib/resty/mtls.lua with the content of this `tls.lua` file. This makes it easier to test for patches, memory leaks...etc.